engine/src/main/resources/application.conf

```diff
@@ -73,6 +73,10 @@ call-caching {
 }
 
 backend {
+  // Use the new route of execution starting ShadowWorkflowActor
+  // Once all features related to pluggable backends are in place, this will be deprecated
+  shadowExecutionEnabled = false
```

Comment:
I wouldn't put this in `backend` stanza because a) I'm in the process of ripping it apart for `BackendConfiguration` and b) this is actually a directive for `WorkflowManagerActor`.  Can I suggest a `system` stanza instead?  I'm already relocating at least one displaced configuration option to that in my branch.